### PR TITLE
Using segmented control for colors page

### PR DESCRIPTION
**Pull request by niels9001**
This PR removes the ToggleButtons with the Segmented control from the Windows Community Toolkit, as per design spec.

SegmentedControl

### CODE REVIEW

File: WinUIGallery/ControlPages/DesignGuidance/ColorsPage.xaml.cs [hunk 1]
```diff
@@ -12,41 +11,34 @@ public sealed partial class ColorsPage : Page
...
+        }
 
-                        button.IsChecked = false;
-                }
-            }
+        private void Page_Loaded(object sender, Microsoft.UI.Xaml.RoutedEventArgs e)
+        {
+            PageSelector.SelectionChanged -= OnSelectionChanged;
+            NavigationFrame.Navigate(typeof(TextSection));
+            PageSelector.SelectionChanged += OnSelectionChanged;
```
Comment (chingucoding) [inline WinUIGallery/ControlPages/DesignGuidance/ColorsPage.xaml.cs:41]: Why is this needed? Especially hooking up the SelectionChanged event?

File: WinUIGallery/WinUIGallery.DesktopWap.csproj [hunk 2]
```diff
@@ -28,7 +28,8 @@
        in the WinUI repo, or the next ItemGroup below when standalone. -->
   <ItemGroup>
     <PackageReference Include="ColorCode.Core" />
-    <PackageReference Include="CommunityToolkit.Labs.WinUI.SettingsControls" Version="0.0.16" />
+    <PackageReference Include="CommunityToolkit.Labs.WinUI.SegmentedControl" Version="0.0.2" />
+    <PackageReference Include="CommunityToolkit.Labs.WinUI.SettingsControls" Version="0.0.17" />
```
Comment (chingucoding) [inline WinUIGallery/WinUIGallery.DesktopWap.csproj:32]: Is this needed for this PR?

Comment (niels9001) [reply WinUIGallery/WinUIGallery.DesktopWap.csproj]: Whoops, pressed the Update all button I guess. Fixed!

File: WinUIGallery/ControlPages/DesignGuidance/ColorsPage.xaml.cs (hunk 1 above)
Comment (niels9001) [reply WinUIGallery/ControlPages/DesignGuidance/ColorsPage.xaml.cs]: Updated, a bit cleaner now :)

**Review by chingucoding — APPROVED** [review]
